packages/cli/package.json

```diff
@@ -63,7 +63,6 @@
     "path": "^0.12.7",
     "prettier": "^2.8.4",
     "prettier-plugin-solidity": "^1.1.2",
-    "solmate": "https://github.com/Rari-Capital/solmate.git#9cf1428245074e39090dceacb0c28b1f684f584c",
```

Comment:
This isn't used in CLI directly, but the CLI does use solecs, which has this as a peer dep (among others that aren't included here), so I'm unsure if this is actually needed.

cc @alvrs @dk1a

Reply:
I'm pretty sure solmate is used exclusively for DSTestPlus in solecs tests, so it's fine to remove in all other packages

Reply:
let's remove it!